src/CodeGeneration.Roslyn.Attributes/CodeGeneration.Roslyn.Attributes.csproj

```diff
@@ -1,14 +1,7 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
 
   <PropertyGroup>
-    <TargetFrameworks>
-      netstandard1.0;
-      netstandard2.0;
-      net40;
-      net20;
-      portable-net40+win8+wpa81;
-      portable-net45+win8+wpa81+wp8
-    </TargetFrameworks>
+    <TargetFrameworks>netstandard1.0;netstandard2.0</TargetFrameworks>
```

Comment:
Why is 2.0 needed? This would double the compile time of this assembly and I can't think of why we'd need it.

Reply:
Yes, the compile time is doubled.

I've kept netstandard2.0, because it has a much smaller dependency footprint for dependents.

Compare
* NETStandard2.0 deps list for v2.0.3: https://www.nuget.org/packages/NETStandard.Library/2.0.3
* NETStandard1.6 deps list for v1.6.1: https://www.nuget.org/packages/NETStandard.Library/1.6.1

Standard2.0 is much more lightweight. However, I'm fine with dropping that target. I just did it because I thought it could provide a better experience for consumers. (I think referencing netstandard2.0 in addition to any 1.x is a kind of "magic" I learned by browsing other projects - it mey very well be a [Cargo Cult](https://en.wikipedia.org/wiki/Cargo_cult) by now!)

Reply:
If the attribute package doesn't actually express that netstandard.library dependency it shouldn't be an issue. And since with the right condition on our attribute even a dependency on itself evaporates, I don't see any advantages to adding a netstandard2.0 target.

Reply:
Ok, I've gone with 1.0 only